controllers/api.js

```diff
@@ -164,6 +166,17 @@ router.post('/auth', postLimiter, async function (req, res) {
   }
 });
 
+router.get('/publicid', postLimiter, async function (req, res) {
+  logger.log('/publicid', [req.id]);
+  let u = new User(redis, bitcoinclient, lightning);
+  if (!(await u.loadByAuthorization(req.headers.authorization))) {
+    return errorBadAuth(res);
+  }
+
+  logger.log('/publicid', [req.id, 'userid: ' + u.getUserId(), 'invoice: ' + req.body.invoice]);
+  res.send(await u.getPublicId());
```

Comment:
Shouldn't you return json here?

Reply:
Also, you might want to return the public id in the response on account creation.